packages/textkit/src/engines/linebreaker/linkedList.js

```diff
@@ -19,6 +19,7 @@ class LinkedList {
     this.head = null;
     this.tail = null;
     this.listSize = 0;
+    this.listLength = 0;
```

Comment:
@diegomura This is an interesting one. TypeScript caught this.listLength being used (see [line 95](https://github.com/diegomura/react-pdf/pull/2514/files#diff-43ed2d44fbfd569877dfaafe36f3f9526ee1b38490aa3a11ce3326cca33338a2R95)) but chances are that it's a bug and it should be listSize instead. Thoughts?